src/state/reducers/index.js

```diff
@@ -63,3 +63,27 @@ export const getLegacyChapterAlignments = (state, chapter) =>
  */
 export const getIsVerseAligned = (state, chapter, verse) =>
   fromAlignments.getIsVerseAligned(state.tool.alignments, chapter, verse);
+
+export function getManifest(state) {
+  return state.projectDetailsReducer.manifest;
+}
+
+export function getProjectSaveLocation(state) {
+  return state.projectDetailsReducer.projectSaveLocation;
+}
+
+export function getContextId(state) {
+  return state.contextIdReducer.contextId;
+}
+
+export function getIsVerseFinished(toolName, state, chapter, verse) {
```

Comment:
this should not be here since it's not processing state but props. Also, we don't need this function anyway. See previous comment where this is used.

Reply:
Done.